### ISSUE TITLE
Is it possible to support scanning multiple codes at one time, and select one for recognition results

### ISSUE THREAD

**Issue by BruceHmy** [markusfisch/BinaryEye#381]
Is it possible to support scanning multiple codes at one time, and select one for recognition results

**Comment by markusfisch** [comment]
Yes, in theory, but only different barcode types (for example a QR Code and an EAN13, but not two QR Codes).

ZXing C++ supports reading multiple (different) barcodes (one of each type), because it can run all decoders in line for a single input image. Reading the same barcode type multiple times would mean processing an input image over and over until no barcode could be found anymore. This would be a very costly operation, of course.

Currently Binary Eye [configures](https://github.com/markusfisch/BinaryEye/blob/master/app/src/main/kotlin/de/markusfisch/android/binaryeye/activity/CameraActivity.kt#L431) ZXing C++ to read just one barcode for performance reasons.
